echo/server/run-worker-cpu.sh

```
@@ -0,0 +1 @@
+dramatiq --queues cpu --processes 2 --threads 8 dembrane.tasks
```

Shellcheck (0.10.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)